### DISCUSSION TITLE
Create a launcher for Leafish

### DISCUSSION THREAD

**Discussion by terrarier2111** [Lea-fish/Leafish#132] [Ideas]
In order to keep everyone updated easily we should probably create a launcher which (on launch) gives the options to either update leafish (if an update is available) or start the game. This would have to be a different program, we could also implement some sort of profile(account) manager for this launcher in the future.

**Comment by PureTryOut** [comment]
Copying my answer from Matrix:

I know applications like Nextcloud just check if an update is available (without a separate launcher or whatever) and if so prompt the user to download it. They then just download an installer which overwrites the existing installation with the new one.
I believe that's how most stuff gets updated on Windows.

For macOS I don't know, and for Linux there is obviously no need for such a thing as it'll be available in package managers once we get our first releases.

I'm not a fan of having a separate launcher at least. For Minecraft it makes sense as you manage versions and stuff through it, but that won't be a thing in Leafish
